--- a/gui/wx.cc
+++ b/gui/wx.cc
@@ -213,10 +213,10 @@
     // know the shortcut, don't annoy them with the message.
     wxString msg;
     msg.Printf(
-      "You have enabled the mouse in Bochs, so now your mouse actions will\n"
+      wxT("You have enabled the mouse in Bochs, so now your mouse actions will\n"
       "be sent into the simulator.  The usual mouse cursor will be trapped\n"
       "inside the Bochs window until you press %s\n"
-      "to turn mouse capture off.", theGui->get_toggle_info());
+      "to turn mouse capture off."), theGui->get_toggle_info());
     wxMessageBox(msg, wxT("Mouse Capture Enabled"), wxOK | wxICON_INFORMATION);
     first_enable = false;
   }
@@ -1028,7 +1028,7 @@
 #endif
 
   wxString msg;
-  msg.Printf("Enable mouse capture\nThere is also a shortcut for this: %s.", theGui->get_toggle_info());
+  msg.Printf(wxT("Enable mouse capture\nThere is also a shortcut for this: %s."), theGui->get_toggle_info());
   theFrame->SetToolBarHelp(ID_Toolbar_Mouse_en, msg);
 
   num_events = 0;
--- a/gui/wxdialog.cc
+++ b/gui/wxdialog.cc
@@ -570,7 +570,7 @@
   buttonSizer = new wxBoxSizer(wxHORIZONTAL);
   mainSizer->Add(logSizer, 0, wxALIGN_CENTER);
   mainSizer->Add(buttonSizer, 0, wxALIGN_CENTER);
-  log = new wxTextCtrl(this, -1, "",
+  log = new wxTextCtrl(this, -1, wxT(""),
       wxDefaultPosition, wxSize(575, 300),
       wxTE_MULTILINE | wxTE_RICH | wxTE_READONLY);
   wxFont font(8, wxFONTFAMILY_TELETYPE, wxFONTSTYLE_NORMAL, wxFONTWEIGHT_NORMAL);
